### PR TITLE
UI fixes and update readme

### PR DESCRIPTION
**Pull request by cp-sumi-k**

### CODE REVIEW

File: vue-frontend/src/pages/jobs/apply/[id].vue [hunk 1]
```diff
@@ -150,7 +150,7 @@
                 class="tw-mb-8 reference-input"
               >
                 <input
-                  class="tw-block tw-w-full tw-border-[1px] tw-border-solid tw-border-[#e2e2e2] tw-rounded-[10px] tw-text-[#3d3d3d] tw-text-[1.125rem] tw-mt-[15px] tw-py-[10px] tw-px-[16px] focus:tw-border-[1px] focus:tw-border-solid focus:tw-border-[#e2e2e2] focus:tw-outline-hidden focus:tw-outline-0 disabled:tw-opacity-[0.8] disabled:tw-cursor-not-allowed reference-by-input tw-pl-0"
+                  class="tw-block tw-w-full tw-border-[1px] tw-border-solid tw-border-[#e2e2e2] tw-rounded-[10px] tw-text-[#3d3d3d] tw-text-[1.125rem] tw-mt-[15px] tw-py-[10px] tw-px-[16px] focus:tw-border-[1px] focus:tw-border-solid focus:tw-border-[#e2e2e2] focus:tw-outline-hidden focus:tw-outline-0 disabled:tw-opacity-[0.8] disabled:tw-cursor-not-allowed reference-by-input"
```
Comment (cp-priyanka-g) [inline vue-frontend/src/pages/jobs/apply/[id].vue:153]: i think text is being cut in size between  320px-336px(mobile view)

Comment (cp-sumi-k) [reply vue-frontend/src/pages/jobs/apply/[id].vue:153]: Hmm, changed the placeholder text for that. Please review @cp-priyanka-g

File: vue-frontend/src/pages/jobs/apply/[id].vue [hunk 2]
```diff
@@ -382,7 +382,7 @@ export default {
         {
           id: 9,
           option: "Other",
-          hint: "Please specify referral source",
+          hint: "Other referral source",
```
Comment (cp-priyanka-g) [inline vue-frontend/src/pages/jobs/apply/[id].vue:385]: you can change the font-size in mobile view rather than changing the text.

Comment (cp-sumi-k) [reply vue-frontend/src/pages/jobs/apply/[id].vue:385]: I think this hint is better than the previous one. Changed text-size

**Review by cp-priyanka-g — APPROVED** [review]

**Pull request merged**